src/main/java/sparrow/data/task/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package sparrow.data.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates task list with operations to be carried out on list.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Constructs an empty task list.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Constructs a task list with the input list.
+     * @param tasks input task list
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
+
+    public void addTask(Task toAdd) {
+        tasks.add(toAdd);
+    }
+    /**
+     * Returns a task list matching the inputted date filter.
+     * @param dateFilter User's command and date to filter by.
```

Comment:
I think you can add a line in between the parameters and the description of the method